core.coffee

```diff
@@ -411,6 +414,20 @@
 			delete: (params) ->
 				return @request(params, method: 'DELETE')
 
+			batch: (data) ->
```

Comment:
For this to work in the supervisor, we'd need a way to either pass the apikey to the request. We've been doing it using `customOptions` which adds it to the query, but it'd be even better to switch to use the Authorization headers. Or should the apikey be passed onto *each* sub-request in the batch? I would assume auth is handled at the batch level and not each particular sub-request.